pkg/maps/multicast/subscribermap_test.go

```diff
@@ -0,0 +1,115 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package multicast
+
+import (
+	"net/netip"
+	"testing"
+
+	"github.com/cilium/ebpf/rlimit"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/cilium/cilium/pkg/bpf"
+	"github.com/cilium/cilium/pkg/testutils"
+)
+
+const (
+	TestGroupV4OuterMapName = "test_cilium_mcast_group_v4_outer"
+)
+
+func TestSubscriberMap(t *testing.T) {
+	testutils.PrivilegedTest(t)
+
+	bpf.CheckOrMountFS("")
+	assert.NoError(t, rlimit.RemoveMemlock())
+
+	groupMapEBPF := NewGroupV4OuterMap(TestGroupV4OuterMapName)
+	err := groupMapEBPF.OpenOrCreate()
+	assert.NoError(t, err)
+
+	t.Cleanup(func() {
+		t.Logf("Cleaning up GroupV4OuterMap")
+		groupMapEBPF.Unpin()
```

Comment:
This can be moved right after `OpenOrCreate()`, most maps don't need to be pinned for the duration of a test. OpenOrCreate() will soon(tm) be changed or removed to require an explicit Pin() instead of having to opt out.